### PR TITLE
Update migration.md

### PR DESCRIPTION
**Pull request by Wenting0905**
PTAL @shenli

### CODE REVIEW

File: op-guide/migration.md [hunk 1]
```diff
@@ -129,8 +129,9 @@ cd mydumper-linux-amd64
 Use the `mydumper` tool to export data from MySQL by typing the following command:
 
 ```bash
-./bin/mydumper -h 127.0.0.1 -P 3306 -u root -t 16 -F 128 -B test -T t1,t2 -o ./var/test
+./bin/mydumper -h 127.0.0.1 -P 3306 -u root -t 16 -F 128 -B test -T t1,t2 --skip-tz-utc -o ./var/test
 ```
+
```
Comment (shenli) [inline op-guide/migration.md]: Remove this line.

**Comment by shenli** [comment]
LGTM
@iamxy PTAL

File: op-guide/migration.md [hunk 2]
```diff
@@ -129,7 +129,7 @@ cd mydumper-linux-amd64
 Use the `mydumper` tool to export data from MySQL by typing the following command:
 
 ```bash
-./bin/mydumper -h 127.0.0.1 -P 3306 -u root -t 16 -F 128 -B test -T t1,t2 -o ./var/test
+./bin/mydumper -h 127.0.0.1 -P 3306 -u root -t 16 -F 128 -B test -T t1,t2 --skip-tz-utc -o ./var/test
```
Comment (siddontang) [inline op-guide/migration.md]: @iamxy I think we should update our mydumper source and enable skip-tz-utc in default.

Comment (ngaut) [inline op-guide/migration.md]: +1

**Comment by ngaut** [comment]
LGTM

**Pull request merged**